modules/ROOT/pages/ai.adoc

```diff
@@ -0,0 +1,102 @@
+= AI Assistant plugin
+:navtitle: AI
+:description: Sends queries to registered AI APIs and sets the results in the document or temporarily stores the results for later use.
+:description_short: Query AI APIs and accept results.
+:keywords: plugin, ai, artificial intelligence, query, search.
+:pluginname: AI Assistant
+:plugincode: ai
+:plugincategory: premium
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+include::partial$misc/admon-requires-6.6v.adoc[]
+
+The {pluginname} allows a {productname} document to send queries to registered AI APIs and either:
+
+. add the returned results to a document; or
+. temporarily store the returned results for later use.
```

Comment:
Is this statement accurate @MitchCTiny , as the returned result is only stored while the editor instance is active, and not retrievable after the editor instance is closed.

> temporarily store the returned results for later use.

Reply:
The results will be discarded on closing the editor and can't retrieved later

Reply:
Need to update the `:description` field as well

Reply:
See [this commit](https://github.com/tinymce/tinymce-docs/commit/bfcd5b106b89d76dc63174364ab46ba650ca9e11) for the re-write that covers this.